### PR TITLE
[ros2topic] Handle multiple namespace parts in message type

### LINKED ISSUE #235: [ros2topic] 'The passed message type is invalid'
## Bug report

**Required Info:**

- Operating System:
  - Ubuntu 18.04 (and presumably all other platforms)
- Installation type:
  - source
- Version or commit hash:
  - tag 0.7.2 (8f419664cd37a42702365c72e88cdbf28ee7b18c)
- DDS implementation:
  - Fast-RTPS
- Client library (if applicable):
  -  rclpy

#### Steps to reproduce issue

Start publishing to a topic:
```
ros2 topic pub /chatter std_msgs/Empty
```

Try using the verbs `bw`, `hz`, and `delay`. E.g.
```
ros2 topic bw /chatter
```

#### Expected behavior

The tools work (e.g. echo bandwidth for `bw`).

#### Actual behavior

All tools log the message **"The passed message type is invalid"** and exit.

#### Additional information

It's likely this is a bug introduced by https://github.com/ros2/ros2/issues/677.

### PR DESCRIPTION
**Pull request by jacobperron**
Fixes #235.
Now the 'bw', 'hz', and 'delay' verbs work again.

### CODE REVIEW

**Comment by jacobperron** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/7101/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/3323/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/5817/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/6921/)

**Comment by mjcarroll** [comment]
> Now the 'bw', 'hz', and 'delay' verbs work again.

Certainly a nice improvement ;)

File: ros2topic/ros2topic/api/__init__.py [hunk 1]
```diff
@@ -57,8 +57,8 @@ def __call__(self, prefix, parsed_args, **kwargs):
 def import_message_type(topic_name, message_type):
     # TODO(dirk-thomas) this logic should come from a rosidl related package
     try:
-        package_name, message_name = message_type.split('/', 2)
-        if not package_name or not message_name:
+        package_name, *message_name = message_type.split('/')
+        if not package_name or not all(message_name):
```
Comment (dirk-thomas) [inline ros2topic/ros2topic/api/__init__.py]: This wouldn't raise anymore if the input string doesn't contain any slashes?

Comment (jacobperron) [reply ros2topic/ros2topic/api/__init__.py]: Ah, good catch. I thought `all` would handle that but doesn't. I can update this to make sure we have at least one part to `message_name`.

Comment (mjcarroll) [reply ros2topic/ros2topic/api/__init__.py]: Ah, didn't catch that, forgot that `all([]) == True`

Comment (jacobperron) [reply ros2topic/ros2topic/api/__init__.py]: Updated edbb844.

File: ros2topic/ros2topic/api/__init__.py [hunk 2]
```diff
@@ -57,8 +57,8 @@ def __call__(self, prefix, parsed_args, **kwargs):
 def import_message_type(topic_name, message_type):
     # TODO(dirk-thomas) this logic should come from a rosidl related package
     try:
-        package_name, message_name = message_type.split('/', 2)
-        if not package_name or not message_name:
+        package_name, *message_name = message_type.split('/')
+        if not package_name or 0 == len(message_name) or not all(message_name):
```
Comment (dirk-thomas) [inline ros2topic/ros2topic/api/__init__.py]: Instead of `0 == len(message_name)` write `not message_name`.

Comment (jacobperron) [reply ros2topic/ros2topic/api/__init__.py]: ead1932

**Pull request merged**